### DISCUSSION TITLE
Podcast Section

### DISCUSSION THREAD

**Discussion by cubansmoke** [dailydotdev/daily#1285] [Feature Requests]
A podcast section for listening to the latest tech news from personally curated sources. Should be able to add custom sources.

**Comment by idoshamun** [comment]
Hi @fingjohnwick,

Where do you usually find podcasts to listen?

**Reply by cubansmoke** [reply]
Spotify mostly.

**Reply by idoshamun** [reply]
Thanks! We will consider it :)
